web/admin/application/controllers/KlearCustomTarificatorController.php

```diff
@@ -0,0 +1,268 @@
+    {
+        if ($this->getParam("tarificate")) {
+            $errors = $this->_getFormErrors();
+            if (!is_null($errors)) {
+                $this->_confirmDialog($errors);
+            } else {
+
+                $container = \Zend_Registry::get('container');
+
+                /** @var DataGateway $dataGateway */
+                $dataGateway = $container->get(DataGateway::class);
+
+                /** @var CompanyBillingService $billingService */
+                $billingService = $container->get(BillingService::class);
+
+                /** @var \Ivoz\Provider\Domain\Model\Company\CompanyDto $companyDto */
+                $companyDto = $dataGateway->find(
+                    \Ivoz\Provider\Domain\Model\Company\Company::class,
+                    $this->getParam("parentId")
+                );
+
+                $callDuration = $this->getParam('duration');
+
+                try {
+                    $response = $billingService->simulateCall(
+                        'b' . $companyDto->getBrandId(),
+                        'c' . $companyDto->getId(),
+                        $this->getParam('number'),
+                        $callDuration
+                    );
+
+                    $this->_helper->log("[Tarificator] Result: Not null");
+                    $message = $this->_getTarificationInfo($response);
+
+                } catch (\Exception $e) {
+
+                    $message = $e->getMessage();
+                    $this->_helper->log("[Tarificator] Result: error " . $message);
+                }
+
+                $title = $this->_helper->translate("Results");
+                $this->_showDialog(
+                    $title,
+                    $message, false,
+                    "Close",
+                    false,
+                    '80%'
+                );
+            }
+
+        } else {
+            $this->_confirmDialog();
+        }
+    }
+
+
+    protected function _confirmDialog ($errorMessage = "")
+    {
+        $title = $this->_helper->translate("Rating Profile Tester");
+        $message = $errorMessage;
+        $message .= "<table class='kMatrix'>";
+        $message .=     "<tr>";
+        $message .=         "<th class='ui-widget-header multiItem notSortable'>";
+        $message .=             $this->_helper->translate("Phone number");
+        $message .=         "</th>";
+        $message .=         "<th class='ui-widget-header multiItem notSortable'>";
+        $message .=             $this->_helper->translate("Duration");
+        $message .=             ' (';
+        $message .=             $this->_helper->translate("seconds");
+        $message .=             ')';
+        $message .=         "</th>";
+        $message .=     "</tr>";
+        $message .=     "<tr>";
+        $message .=         "<td class='ui-widget-content'>";
+        $message .=             '<input type="text" name="number" placeholder="+34123456789" class="ui-widget ui-state-default ui-corner-all"';
```

Comment:
For a future enhancement, It would be nice if the default value uses the `$brand->getCountry()->getCountryCode()` as prefix instead of _+34_

That way the format will be automatically understanded by the administator at first glance